en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md

```diff
@@ -0,0 +1,209 @@
+---
+id: network-dell-os10-snmp
+title: Dell OS10 SNMP
+---
+
+## Overview
+
+Dell OS10 is the new operating system for Dell network equipment. It combines the best of Linux, open computing and networking to advance open networking disaggregation.
+This Plugin-Pack monitors all physical and system points using SNMP.
```

Comment:
You can simply remove this part, Plugin-Pack Assets / Monitored Objects is enough

Reply:
Done